packages/server/postgres/queries/src/getPatientZeroByDomainQuery.sql

```diff
@@ -2,4 +2,4 @@
   @name getPatientZeroByDomainQuery
```

Comment:
+1 rename this query and the corresponding .ts code, as it no longer returns the patient 0